docs/index.html

```diff
@@ -8,6 +8,11 @@
     <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
     <link rel="icon" type="image/png" href="./favicon.png">
     <script type="module" src="https://unpkg.com/rapidoc/dist/rapidoc-min.js"></script>
+    <script type="text/javascript">
```

Comment:
Isn't this file auto-generated by the build tool?

Reply:
not anymore, that was using redocly, only the `docs/dist.yaml` is compiled now